libraries/botbuilder-core/tests/test_bot_framework_adapter.py

```diff
@@ -247,3 +254,31 @@ def test_should_create_connector_client(self):
             )
 
         await adapter.process_activity(incoming, "", aux_func_assert_tenant_id_copied)
+
+    async def test_should_create_valid_conversation_for_msteams(self):
+
+        tenant_id = "testTenant"
+
+        reference = deepcopy(REFERENCE)
+        reference.conversation.tenant_id = tenant_id
+        reference.channel_data = {"tenant": {"id": tenant_id}}
+        adapter = AdapterUnderTest()
+
+        async def aux_func_assert_valid_conversation(context):
```

Comment:
Can you add a called boolean variable like in test_should_process_activity to make sure you're executing this assertions? Thanks!